django/contrib/gis/db/models/fields.py

```diff
@@ -444,4 +443,5 @@ def get_transform(self, name):
             )
         except ValueError:
             pass
-        return super().get_transform(name)
+        else:
```

Comment:
Why is this changed? I think replacing `pass` with `return None` would be more explicit if that's really the correct logic.